frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -2289,7 +2289,7 @@ function ReaderHighlight:onMoveHighlightIndicator(args)
                 -- quadruple press: 64 single distances, almost move to screen edge
                 if G_reader_settings:nilOrTrue("highlight_non_touch_spedup") then
                     -- user selects whether to use 'constant' or [this] 'sped up' rate (speed-up on by default)
-                    local x_inter = G_reader_settings:readSetting("highlight_non_touch_interval")
+                    local x_inter = G_reader_settings:readSetting("highlight_non_touch_interval", 1)
```

Comment:
```suggestion
                    local x_inter = (G_reader_settings:readSetting("highlight_non_touch_interval") or 1)
```

Reply:
You don't need the parens here (I put them for the previous example, which was part of arithmetic computation.

Reply:
silly me